install/minikube-installation-guide.md

```diff
@@ -0,0 +1,240 @@
+| `--bootstrapper=kubeadm` | As recommended for [minikube CRI-o](https://kubernetes.io/docs/setup/minikube/#cri-o) |
+
+> **Note:** This example deploys Kata Containers using CRI-O. The same procedure should also work
+> for a `containerd` based Kubernetes installation, but is not covered in this document.
+
+> **Note:** Prior to Minikube/Kubernetes v1.14, the beta `RuntimeClass` feature also needed enabling with
+> the following. Recent Minikube/Kubernetes installations should no longer require this setting.
+>
+> | what | why |
+> | ---- | --- |
+> | `--feature-gates=RuntimeClass=true` | Kata needs to use the RuntimeClass Kubernetes feature |
+
+The full command is therefore:
+
+```sh
+$ minikube start --vm-driver kvm2 --memory 6144 --network-plugin=cni --enable-default-cni --container-runtime=cri-o --bootstrapper=kubeadm 
+```
+
+> **Note:** For Kata Containers >v1.6.2, the now default `tcfilter` networking of Kata Containers
+> did not work for Minikube versions <v1.1.1. Please ensure you use Minikube version >=v1.1.1.
+
+## Checking for nested virtualization
+
+For Kata Containers to work under a Minikube VM, your host system must support
+nested virtualization. If you are using a Linux system utilising Intel VT-x 
+and the `kvm_intel` driver, you can perform the following check:
+
+```sh
+$ cat /sys/module/kvm_intel/parameters/nested
+Y
+```
+
+If your system does not report `Y` from the `nested` parameter, then details on how
+to enable nested virtualisation can be found on the
+[KVM Nested Guests page](https://www.linux-kvm.org/page/Nested_Guests)
+
+## Check Minikube is running
+
+Before you install Kata Containers, perform a quick check that your Minikube is
+operating. On your guest:
+
+
+```sh
+$ kubectl get nodes
+NAME       STATUS   ROLES    AGE   VERSION
+minikube   Ready    master   78m   v1.13.4
+```
+
+Check you have virtualization enabled inside your Minikube:
+
+```sh
+# No color option, as minikube is running busybox
+$ minikube ssh "egrep 'vmx|svm' /proc/cpuinfo"
+
+# if you get a long line of output here, you have it enabled!
+flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon rep_good nopl xtopology cpuid pni pclmulqdq vmx ssse3 fma cx16 pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch cpuid_fault invpcid_single pti retpoline rsb_ctxsw tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt xsaveopt xsavec xgetbv1 xsaves arat umip
+
+# A failed result means it is not enabled:
+ssh: Process exited with status 1
+```
+
+## Installing Kata Containers
+
+You can now install the Kata Containers runtime components. You will need a local copy of some Kata
+Containers components to help with this, and then use the host `kubectl` (that Minikube has already
```

Comment:
> the host `kubectl`

Why is "host" mentioned here? Are there more than one versions of `kubectl`?

Reply:
reworded. It is not that there is more than one `kubectl` so much, in as much that you need to run `kubectl` *on* the host, and not try to run it inside minikube.